py/client-ticking/README.md

```diff
@@ -11,6 +11,9 @@ are working on unifying these two libraries.
 
 ## Prerequisites
 
+# *CRISTIAN*: need python3 installed and specifically the package python3-dev ???
```

Comment:
Yes, python3-dev contains header files for interfacing with python, which cython is going to depend on.

Reply:
We also need `python3-venv` for the `python -m venv` command mentioned below.

Reply:
I have a proposal with some fixes for README.md here: https://github.com/kosak/deephaven-core/pull/15
With those changes the instructions worked for me.

Reply:
Other OSes (including other linuxes) are outside the scope of this specific PR. I do think we'll support them eventually.